doc.json

```diff
@@ -442,7 +442,99 @@
           "description": "Permission denied, user cannot rate their own article"
           }
         }
+      }
+    },
+      "/likeArticle/:slug": {
+      "post": {
+        "tags": [
+          "Articles"
+        ],
+        "summary": "Like an article",
+        "description": "Like an article",
+        "operationId": "likeArticle",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "authorization",
+            "in": "header",
+            "description": "A token to verify the user",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "like",
+            "description": "Like an article",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/rating"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Article liked successfully"
+          },
+          "404": {
+            "description": "This article does not exist"
+          },
+          "200": {
+          "description": "Article has unliked successfully"
```

Comment:
From your likeArticle controller, the message for a successful unliking of an article is different from the one you have here. `message: 'You have unliked this article'`